extension/background.js

```diff
@@ -55,105 +55,51 @@ async function getA11yScores() {
-                  } else {
-                    // This should never happen but we should be prepared to handle this.
-                    linkText.innerHTML =
-                      `Ha11y Error ` +
-                      originalHTML +
-                      ` - Ha11y error was ${json.errors}`;
-                  }
-                })
-                .catch((error) => {
-                  //console.log(error);
-                  linkText.innerHTML =
-                    "Ha11y Error " +
-                    originalHTML +
-                    ` - Ha11y Server Error was ${error.toString()}`;
-                });
+              linkTextElement = linkElement.getElementsByClassName("mCBkyc")[0];
             } else if (
               linkElement.classList.contains("l") ||
               linkElement.classList.contains("fl") ||
               container.classList.contains("HiHjCd")
             ) {
-              let storedLinkElement = linkElement;
-              let originalHTML = linkElement.innerHTML;
-
-              storedLinkElement.innerHTML = "Ha11y Loading - " + originalHTML;
-              fetch(apiUrl)
-                .then((results) => {
-                  return results.json();
-                })
-                .then((json) => {
-                  if (json.totalScore) {
-                    // Temporary hack for displaying the results from the API
-                    storedLinkElement.innerHTML =
-                      `Ha11y Score ${json.totalScore} - ` + originalHTML;
-                  } else {
-                    // This should never happen but we should be prepared to handle this.
-                    storedLinkElement.innerHTML =
-                      `Ha11y Error ` +
-                      originalHTML +
-                      ` - Ha11y error was ${json.errors}`;
-                  }
-                })
-                .catch((error) => {
-                  //console.log(error);
-                  storedLinkElement.innerHTML =
-                    "Ha11y Error " +
-                    originalHTML +
-                    ` - Ha11y Server Error was ${error.toString()}`;
-                });
+              linkTextElement = linkElement;
             } else if (linkElement.hasAttribute("data-ved")) {
-              let linkText = linkElement.getElementsByTagName("h3")[0];
-              let originalHTML = linkText.innerHTML;
-              linkText.innerHTML = "Ha11y Loading - " + originalHTML;
-              // The following line should be considered make the app messier rather than more beneficial
-              //linkText.setAttribute("aria-live","polite");
+              linkTextElement = linkElement.getElementsByTagName("h3")[0];
+            }
+            let originalHTML = linkTextElement.innerHTML;
+            linkTextElement.innerHTML = "Ha11y Loading - " + originalHTML;
+            // The following line should be considered make the app messier rather than more beneficial
```

Comment:
Is this needed? If so, might consider rewording this.

Reply:
that line is in a bunch of places